mono/metadata/sysmath.c

```diff
@@ -108,27 +54,27 @@ ves_icall_System_Math_Sin (gdouble x)
 }
 
 gdouble 
-ves_icall_System_Math_Cos (gdouble x)
+ves_icall_System_Math_Cbrt (gdouble x)
```

Comment:
did you sort the functions by name?

Reply:
um.. yes, the list looks sorted (also no warrnings about the sort order)

Reply:
FYI, we don't do this sort of code shuffling in runtime code. We favor smaller diffs.

Reply:
ah, sorry, I thought you mean icall-defs order. Re-order in this file was made by a mistake